src/javascript/app/pages/trade/charts/digit_info.js

```diff
@@ -108,6 +109,8 @@ const DigitInfo = (() => {
         $('#digit_underlying').html($(elem)).val(underlying);
         $('#digit_domain').text(domain.charAt(0).toUpperCase() + domain.slice(1));
         $('#digit_info_underlying').text($('#digit_underlying option:selected').text());
+        Dropdown('#digit_underlying');
+        Dropdown('#tick_count');
```

Comment:
couldn't we add two selectors at the same time? have you tried `Dropdown('#digit_underlying, #tick_count');`? theoretically jquery should be able to handle this, not sure about any exceptions in the code that would break with this